src/TouristInfoCards.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  return (
+    <div className="container-tourist">
+      <div className="card">
+        <img
+          src="https://peoplemakeglasgow.com/templates/people/images/logo.png"
```

Comment:
Rather than putting the data directly into these cards could extra in to a file similar to `FakeBookings.js` and this pass it into this component as a `prop`?